### ISSUE TITLE
Preview of SVG files do not render.

### ISSUE THREAD

**Issue by victorhvs-iabcd** [safrazik/vue-file-agent#109]
**Describe the bug**
SVG preview not rendering.

**To Reproduce**
1. Select a svg file
2. Look at preview file

**Expected behavior**
I expect that vue-file-agent could render SVG files the same as PNG files for example.

**Screenshots**
![Captura de Tela 2020-11-17 às 5 04 51 PM](https://user-images.githubusercontent.com/53314903/99441882-4b486180-28f7-11eb-872e-47ace8638a75.png)

![Captura de Tela 2020-11-17 às 5 08 05 PM](https://user-images.githubusercontent.com/53314903/99442029-7f238700-28f7-11eb-8117-c28c80fb1a8c.png)

**Desktop (please complete the following information):**
 - OS: MacOS Catalina
 - Browser: Firefox
 - Version : 82

**Comment by safrazik** [comment]
Yes, SVG file previews are supported already. Can you please try in a different browser or with a different file? If only the file of concern is not showing the preview, can you please reply with information about the file (e.g: file size, etc)
